### ISSUE TITLE
Branding?

### ISSUE THREAD

**Issue by sk1p** [LiberTEM/LiberTEM-blobfinder#8]
Documentation may need:

- [ ] favicon
- [ ] logo

Maybe we can have a common logo/icon for "LiberTEM module", as a kind of placeholder, as long as the modules themselves don't have their own?

**Comment by uellue** [comment]
Yes, I'd say we have two brands: LiberTEM and Ptychography 4.0. We just have to make sure that users understand if they are on the main documentation or on a sub-package documentation.

**Comment by uellue** [comment]
Postponing to 0.5 per https://github.com/LiberTEM/LiberTEM-blobfinder/issues/7#issuecomment-584594098
